Gemfile

```diff
@@ -26,6 +26,7 @@ gem "dry-transaction", ">= 0.10.0"
 gem "dry-types"
 gem "dry-validation"
 gem "dry-view", ">= 0.3.0"
+gem "redcarpet" # Markdown rendering
```

Comment:
fyi, not important, GitHub recently switched to use https://github.com/gjtorikian/commonmarker. ([html-pipeline](https://github.com/jch/html-pipeline)'s markdown bundled gem).